deflect/StreamSendWorker.h

```diff
@@ -62,20 +63,32 @@ class StreamSendWorker
     ~StreamSendWorker();
 
     /** Enqueue an image to be send during the execution of run(). */
-    Stream::Future enqueueImage(const ImageWrapper& image);
+    Stream::Future enqueueImage(const ImageWrapper& image, bool finish);
+
+    Stream::Future enqueueFinish(); //!< Enqueue a finishFrame()
```

Comment:
This is new to me though, I've never used that anywhere ;-) what rule do you apply to choose between //!< and /** ... */, you check if it can fit on one line?  It just does not look very consistent to me when I look at the entire file but I can live with it if you like it :-)